libs/upgradephp/upgrade.php

```diff
@@ -107,184 +112,194 @@
  * @return bool           TRUE on success, FALSE on failure
  */
 if (in_array('mysqli', @get_loaded_extensions()) && !function_exists('mysqli_set_charset')) {
-	function mysqli_set_charset($link, $charset)
-	{
-		return mysqli_query($link, "SET NAMES '$charset'");
-	}
+    function mysqli_set_charset($link, $charset)
+    {
+        return mysqli_query($link, "SET NAMES '$charset'");
+    }
 }
 
 /**
  * parse_ini_file() replacement.
  * Behaves like parse_ini_file($filename, $process_sections);
  *
- * @author Andrew Sohn <asohn (at) aircanopy (dot) net>
- * @author anthon (dot) pang (at) gmail (dot) com
- *
  * @param string $filename
  * @param bool $process_sections (defaults to false)
  * @return array
+ * @author anthon (dot) pang (at) gmail (dot) com
+ *
+ * @author Andrew Sohn <asohn (at) aircanopy (dot) net>
```

Comment:
moving the author blocks and adding an empty line between them doesn't make things better...